### PR TITLE
Add pathauto and redirect modules

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Install both modules by default, along with settings config.
Updates features with their respective pathauto rule.
Adds install modules code to current hook_update.
Feature revert call should add pathauto to existing site content types.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | yes/no
| Did you perform browser testing? | yes
| Risk level | Low
| Relevant links | https://thinkoomph.jira.com/browse/RIG-123

### CODE REVIEW

File: ecms_base/features/custom/ecms_basic_page/config/install/pathauto.pattern.pages.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+uuid: ba4c50e8-485c-4c49-b288-5f3873996fe4
```
Comment (pfrilling) [inline ecms_base/features/custom/ecms_basic_page/config/install/pathauto.pattern.pages.yml:1]: Should the UUID be here?

File: ecms_base/features/custom/ecms_event/config/install/pathauto.pattern.events.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+uuid: 723dc7d9-aefb-499e-80da-a60f4fd8cd00
```
Comment (pfrilling) [inline ecms_base/features/custom/ecms_event/config/install/pathauto.pattern.events.yml:1]: Same here and the rest of the configuration for pathauto below.

**Review by pfrilling — COMMENTED** [review]
Should the UUID's be included in the pathauto configuration files within the features?

**Comment by ben-hamelin** [comment]
@pfrilling Removed UUIDs and rebased, ready for reivew

**Review by pfrilling — APPROVED** [review]
Looks good, just needs a rebase.

**Comment by ben-hamelin** [comment]
@pfrilling rebased, ready for final review

**Review by pfrilling — APPROVED** [review]
Looks good.

**Comment by ben-hamelin** [comment]
... and again :)

**Pull request merged**